api/src/pcapi/domain/booking_recap/booking_recap.py

```diff
@@ -38,6 +38,7 @@ def __init__(
         booking_confirmation_date: datetime | None,
         booking_is_external: bool,
         booking_amount: float,
+        booking_price_category_label: str,
```

Comment:
```suggestion
        booking_price_category_label: str | None,
```
Vu que ça peut être null dans le dto non ?